### PR TITLE
Release v0.14.6

### PR DESCRIPTION
**Pull request by FloatXD**
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+- [e2e] upgrade $path in e2e test #1502 (@FloatXD )
+- [e2e] upgrade e2e test  #1504 (@FloatXD )
+- [e2e]open k8s.129 ad test #1509 (@FloatXD )
+- [test]add static volume e2e test #1525 (@FloatXD )
+- [test]upgrade api test #1528 (@FloatXD )
+- [e2e]add full test in pr test #1538 (@FloatXD )
+## Documentation
+- docs: update serial check command #1506 (@SSmallMonster )
+- update docs Cached Volumes #1510 (@peng9808 )
+- update docs Cached Volumes #1511 (@peng9808 )
```
Comment (SSmallMonster) [inline CHANGELOG.md:23]: can these be merged to single note?

**Review by SSmallMonster — APPROVED** [review]
over all LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (FloatXD) [reply CHANGELOG.md]: done

**Comment by SSmallMonster** [comment]
lgtm

**Pull request merged**